docker/docker-compose-base.yml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016, EMC, Inc.
+
+version: '2'
+
+volumes:
+  dhcp-leases:
+    external: false
+
+services:
+
+  mongo: # 27017
+    image: mongo:latest
+    network_mode: "host"
+    privileged: true
+
+  files:
+    build: "./files"
+    image: "rackhd/files"
+    network_mode: "host"
+    privileged: true
+    volumes:
+      - "./files/mount:/RackHD/files"
+
+  # RackHD services
+
+  dhcp-proxy: # 68/udp, 4011
+    build: "../on-dhcp-proxy"
+    depends_on:
+      - mongo
+    image: rackhd/on-dhcp-proxy:latest
+    network_mode: "host"
+    privileged: true
+    volumes:
+      - "dhcp-leases:/var/lib/dhcp"
+      - "./monorail:/opt/monorail"
+
+  http: # 9090, 9080
```

Comment:
Technically you would want to build `on-core` and `on-tasks` before building the other services. However I don't think it is important to include them in this docker-compose file.